packages/codemirror-extension/src/commands.ts

```diff
@@ -131,5 +153,177 @@ export const commandsPlugin: JupyterFrontEndPlugin<void> = {
       },
       isEnabled
     });
+
+    app.commands.addCommand(CommandIDs.foldCurrent, {
+      label: trans.__('Fold Current Region'),
+      execute: () => {
+        const notebook = tracker.currentWidget?.content;
+        if (!notebook || !notebook.activeCell) {
+          return;
+        }
+        notebook.mode = 'edit';
+        notebook.activeCell.editor?.focus();
```

Comment:
Can we make it work in file editor too? Also a minimal Galata test might be useful.